### PR TITLE
[EOC-56] create 404 page and set up redirection

### PR DESCRIPTION
**Pull request by alemikolo**
screenshot-localhost-3000-2019 03 11-12-43-13

### CODE REVIEW

File: client/src/app/components/Layout/Layout.jsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ import { UserPropType } from 'common/constants/propTypes';
 import { getCurrentUser } from 'modules/authorization/model/selectors';
 import Footer from '../Footer';
 import Notifications from 'modules/notification';
+import PageNotFound from 'common/components/PageNotFound';
```
Comment (adamklepacz) [inline client/src/app/components/Layout/Layout.jsx]: consider renaming to Page404

File: client/src/common/components/PageNotFound.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React, { Fragment } from 'react';
+import { Link } from 'react-router-dom';
+
+import Toolbar from 'common/components/Toolbar';
+
+const PageNotFound = () => (
+  <Fragment>
```
Comment (adamklepacz) [inline client/src/common/components/PageNotFound.jsx]: you don't need Fragment here

File: client/src/common/components/PageNotFound.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
+.page-not-found {
+  position: absolute;
+  width: 100%;
+  height: 100%;
```
Comment (adamklepacz) [inline client/src/common/components/PageNotFound.scss]: you don't need these 3 properties

File: client/src/common/utils/fetchMethods.js [hunk 4]
```diff
@@ -2,13 +2,19 @@ import {
...
 const handleFetchErrors = resp => {
   if (resp.status === 403) {
     setTimeout(() => {
       window.location = FRONTEND_URL;
     }, NOTIFICATION_TIMEOUT);
   }
+
+  if (resp.status === 404) {
+    history.replace('/page-not-found');
+  }
```
Comment (marekrozmus) [inline client/src/common/utils/fetchMethods.js]: add `return` just in case

**Pull request merged**